src/main/java/com/nirima/jenkins/plugins/docker/DockerCloud.java

```diff
@@ -67,6 +69,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DockerCloud.class);
 
+    static final DockerNodeFactory NODE_FACTORY = DockerNodeFactory.getInstance();
```

Comment:
If this doesn't need to be accessed from outside this class, it should be private.
If it does need to be accesserd from outside this class, it should be `@Restricted(NoExternalUse.class)` to prevent its usage from outside the plugin.